File: src/front-end/CryptExFrontEnd/src/app/deposit-withdraw/models/fiat-withdrawal-view-model.ts
import { WalletViewModel } from "src/app/user/models/wallet-view-model";
import { BankAccountViewModel } from "./bank-account-view-model";
import { PaymentStatus } from "./deposit-view-model";

export interface FiatWithdrawalViewModel {
    id: string;

    amount: number;

    status: PaymentStatus;

    date: string;

    bankAccount: BankAccountViewModel;

    wallet: WalletViewModel;
}

File: src/back-end/CryptEx/CryptExApi/Models/ViewModel/Payment/FiatWithdrawalViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CryptExApi.Models.ViewModel.Payment
{
    public class FiatWithdrawalViewModel
    {
        public Guid Id { get; set; }

        public decimal Amount { get; set; }

        public PaymentStatus Status { get; set; } = PaymentStatus.NotProcessed;

        public DateTime Date { get; set; }

        public BankAccountViewModel BankAccount { get; set; }

        public WalletViewModel Wallet { get; set; }
    }
}
